proto/catalog.proto

```diff
@@ -141,6 +144,7 @@ message SinkFormatDesc {
   plan_common.EncodeType encode = 2;
   map<string, string> options = 3;
   optional plan_common.EncodeType key_encode = 4;
+  map<string, secret.SecretRef> secret_refs = 5;
```

Comment:
Isn't it duplicate with the pre-defined one in sink proto?

Reply:
ditto, not sure about we need to introduce format-encode ones for Sink, cc @xiangjinwu

Reply:
Sink also uses schema.registry, so I guess it need secrets